wremnants/CardTool.py

```diff
@@ -597,6 +598,112 @@ def writeForProcesses(self, syst, processes, label, check_systs=True):
         if syst != self.nominalName:
             self.fillCardWithSyst(syst)
 
+    def setSimultaneousABCD(self):
```

Comment:
I think this would make more sense in the combine_helpers script, the idea being to separate out general features of the combine cards vs. analysis-specific things.